packages/cozy-intent/src/api/models/applications.ts

```diff
@@ -0,0 +1,20 @@
+/** App's description resulting of its manifest.webapp file */
+export interface AppManifest {
+  /** The app's slug */
+  slug: string
+
+  /** The app's mobile information */
+  mobile: AppManifestMobileInfo
+}
+
+/** App's mobile information. Used to describe the app scheme and its store urls */
+export interface AppManifestMobileInfo {
+  /** The app's URL scheme */
+  schema: string
+
+  /** The app's id on Google PlayStore */
+  id_playstore: string
+
+  /** The app's id on Apple AppStore */
+  id_appstore: string
+}
```

Comment:
This should definitively be described to https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/manifest.js#L1 

When a lib deal with "Cozy's generic type", the type should not be defined in the lib, but instead in CozyClient, no?

Reply:
Oh I was not aware of this file. I'll move the def in cozy-client later today 👍